smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -139,6 +142,7 @@ abstract class HTTPBindingProtocolGenerator(
     override var serviceErrorProtocolSymbol: Symbol = ClientRuntimeTypes.Http.HttpError
 
     override fun generateSerializers(ctx: ProtocolGenerator.GenerationContext) {
+//        if (usesSchemaBasedSerialization(ctx)) return // temporary condition
```

Comment:
Is this and 3 more below commented lines with same condition-based skip supposed to be commented out?

Reply:
Yes, these are commented to allow for enabling & disabling schemas and current serde for testing.